src/cases/containers/query-management-container/query-management-container.component.ts

```diff
@@ -174,6 +175,14 @@ export class QueryManagementContainerComponent implements OnInit, OnDestroy {
     this.showConfirmation = true;
   }
 
+  public callbackConfirmationBody(text: string): void {
+    if (text) {
+      this.callbackConfirmationBodyText = text;
+    } else {
+      this.callbackConfirmationBodyText = 'Our team will read your query and respond.';
```

Comment:
Should probably put text in a constant so it can be referred to in the spec file without changing every piece of text